src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -97,9 +99,14 @@ TriggeredSubscription::~TriggeredSubscription()
 * TriggeredSubscription class is shared for NGSI9 and NGSI10 subscriptions, so it is better
 * to keep expressions (an artifact for NGSI10) out of the constructor, in its independent fill
 * method
-*
 */
-void TriggeredSubscription::fillExpression (const std::string& q, const std::string& georel, const std::string& geometry, const std::string& coords)
+void TriggeredSubscription::fillExpression
+(
+  const std::string&  q,
+  const std::string&  georel,
+  const std::string&  geometry,
+  const std::string&  coords
+)
 {
```

Comment:
> TriggeredSubscription is older than subCache and should probably have changed when the subCache was implemented.

However, note that we also have a `-noCache` way of working, so probably keep TriggeredSubscription idependent from the csub cache is a good idea.